Dockerfile

```diff
@@ -25,35 +27,33 @@ RUN cp -r build/* /app/static
 RUN mv /app/static/static/* /app/static/
 RUN rm -r /app/static/static
 
+# Set up Django
+FROM python:alpine
```

Comment:
With this alpine container specifically, how much does it actually decrease the image size and does it have any impact on running time?

I only ask, because I know I've hit a few issues issues with it before on alpine images [this explains the issue decently well](https://pythonspeed.com/articles/alpine-docker-python/).  The only thing that article doesn't explain is that this is really only an issue that's brought up with compiled code that has a python wrapper (you can ignore the zealous claim about don't use alpine for python period) .  Looking at our requirements, I believe most of what we have is pure python so it shouldn't cause an issue for the most part.  I think `psycopg2` is the only non-pure python module I saw.  
Probably won't be an issue, but is something to be aware of.

Reply:
I actually saw the same article. 😛 
Yeah, psycopg2 is the only compiled module.

For comparison, the image is **284 MB** when using Alpine, and **984 MB** when using the 'default' Python image (based off of `buildpack-deps`).  That said, the Alpine version does take slightly longer to build, but I think the smaller image size in this case is worth it.